config/tailwind.config.js

```diff
@@ -1,4 +1,4 @@
-const defaultTheme = require('tailwindcss/defaultTheme')
+// // const defaultTheme = require('tailwindcss/defaultTheme')
```

Comment:
If we are not using defaultTheme then we can remove it and the `theme` key below?

Reply:
it's a template - I'm not sure if that's gonna be used or not in the branched app